tools/ci_build/github/azure-pipelines/post-merge-jobs.yml

```diff
@@ -112,7 +94,13 @@ stages:
       ORT_EP_NAME: CPU
       GenerateDocumentation: false
       EnablePython: false
-      MachinePool: 'Win-CPU-2021'
+      # The public ADO project
+      ${{ if eq(variables['System.CollectionId'], 'f3ad12f2-e480-4533-baf2-635c95467d29') }}:
+        MachinePool: 'Win-CPU-2019'
+      # The private ADO project
+      ${{ if eq(variables['System.CollectionId'], 'bc038106-a83b-4dab-9dd3-5a41bc58f34c') }}:
+        MachinePool: 'Win-CPU-2021'
```

Comment:
Same question here. Should there be a default pool or error message if neither CollectionId is found?

Reply:
If not found, you would not be able to queue a build since no pool was specified.  Unlikely we will have a different ADO account other than onnxruntime and aiinfra.

Reply:
Maybe it would be clearer that these values weren't expected to ever change if they mentioned https://dev.azure.com/onnxruntime and https://dev.azure.com/aiinfra instead of just saying 'public ADO' and 'private ADO' which provides no real context (at least not to me) as to what the values represent. It's not like the latter is secret given we publish public feeds from it.

Reply:
Is it possible to default the value of MachinePool to something like `MachinePool: 'Could not set machine pool. Unexpected value in System.CollectionId of ...'` so its slightly more obvious why it's broken?

Reply:
Names of the machines pool are unique. One pool cannot exist in two places. I can make it more obvious by changing the  names of the machine pools.